app/descriptor/views.py

```diff
@@ -0,0 +1,8 @@
+from flask import render_template
+from flask.ext.login import login_required
+from . import descriptor
+
+@descriptor.route('/')
+@login_required
+def view():
```

Comment:
There isn't actually more than one `index` - there's `main.index`, `admin.index`, `descriptor.index`... They're all different :) The naming convention `index` just indicates that this is the "main" page for that blueprint.